src/components/QueryAny/index.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Query from '../Query';
 
+/**
+* The QueryAny predicate takes an array of values. It works exactly the same
+* way as the at operator, but checks whether the fragment matches any of the values in the array.
```

Comment:
The QueryAny component ... as the QueryAt component ...
here we are converting this operators to component so talk in term of component